infra/app/web.bicep

```diff
@@ -57,6 +57,16 @@ module web '../core/host/appservice.bicep' = {
   }
 }
 
+// Storage Blob Data Contributor
+module storageBlobRoleWeb '../core/security/role.bicep' = if (authType == 'rbac') {
```

Comment:
I suspect that this may be the cause of #643, and it was failing silently until #706, where it no longer fails silently